app/Http/Controllers/Admin/UserController.php

```diff
@@ -323,6 +323,50 @@ public function regen_apikey(int $id, Request $request): RedirectResponse
         return redirect(route('admin.users.edit', [$id]));
     }
 
+    public function verify_email(int $id, Request $request): RedirectResponse
```

Comment:
Are sure these should be in the admin route only? Seems like they should also be in the frontend user controller

Reply:
Nevermind, I see what these are for

Reply:
These methods are just to allow an admin to reset the verification of an email (this way the user has to verify their email again) or to verify a user's email manually without them having to do i. These are functionalities which have been requested in the issue